spec/index.bs

```diff
@@ -10248,21 +10253,16 @@ GPUComputePassEncoder includes GPUBindingCommandsMixin;
 ### Compute Pass Encoder Creation ### {#compute-pass-encoder-creation}
 
 <script type=idl>
-enum GPUComputePassTimestampLocation {
-    "beginning",
-    "end"
-};
-
-dictionary GPUComputePassTimestampWrite {
+dictionary GPUComputePassTimestampWrites {
     required GPUQuerySet querySet;
-    required GPUSize32 queryIndex;
-    required GPUComputePassTimestampLocation location;
+    GPUSize32 beginningOfPassWriteIndex;
+    GPUSize32 endOfPassWriteIndex;
```

Comment:
These names are very verbose, especially when viewed in context:

```js
encoder.beginRenderPass({
  colorAttachments: attachments,
  timestampWrites: {
    querySet: timestampQueries,
    beginningOfPassWriteIndex: 0,
    endOfPassWriteIndex: 1,
  }
});
```

Given that they're already part of a pass, and already under a key called `timestampWrites`, what about something like `beginIndex` and `endIndex`?

Reply:
I have what is probably a dumb question but ... is there a point to being able to specify a begin and end index separate vs just specifying 1 and having it takes 2 slots (and maybe required it to be a align(offset, 2)?`. Is it common to want to put these 2 stamps at unique places?

Advantages

* less to type?
* less to validate?
* simpler to use?

Reply:
Theoretically one might use the beginning of one pass and the end of a different, later pass. In practice all of these APIs work for an application though, I'd only maybe worry about abstractions on top of WebGPU being limited by our API.

Reply:
The current design makes it possible to only write one of the two indices